crates/rpc/src/v06.rs

```diff
@@ -34,6 +34,7 @@ pub fn register_routes() -> RpcRouterBuilder {
         .register("starknet_addDeployAccountTransaction"     , method::add_deploy_account_transaction)
         .register("starknet_addInvokeTransaction"            , method::add_invoke_transaction)
         .register("starknet_estimateFee"                     , method::estimate_fee)
+        .register("starknet_estimateMessageFee"              , method::estimate_message_fee)
```

Comment:
This is already registered higher up. The register function currently just over-writes it instead of panic'ing which it probably should do instead.

Reply:
Good catch, I forgot to remove that...